control-plane/api-gateway/binding/route_binding.go

```diff
@@ -163,6 +163,31 @@ func (r *Binder) bindRoute(route client.Object, boundCount map[gwv1beta1.Section
 			parent:  ref,
 			results: result,
 		})
+
+		httproute, ok := route.(*gwv1beta1.HTTPRoute)
+		if ok {
+			if !externalRefsOnRouteAllExist(httproute, r.config.Resources) {
+				results = append(results, parentBindResult{
+					parent: ref,
+					results: []bindResult{
+						{
+							err: errExternalRefNotFound,
```

Comment:
I don't recall what happens when you have two binding results for the same parent - do the statuses/syncing still work out if the route successfully bound as per above and then has an error here? I'd assume it's an ordering thing and whatever the last bind result is for the parent is what's actually used?

Reply:
I believe its the last bind result, yes.